addons/udes_stock_cron/models/stock_picking.py

```diff
@@ -95,8 +95,8 @@ def _reserve_stock_assign(self):
         """
         # Assign at the move level disabling refactoring
         moves = self.move_lines
-        moves._action_assign()
-        return moves.picking_id
+        refactored_moves = moves._action_assign()
```

Comment:
I wouldn't call this one refactored, since there could be other code in the future that splits/changes/returns different recordset of moves.